src/main/java/org/folio/inventory/resources/AbstractInstances.java

```diff
@@ -365,6 +481,32 @@ protected CollectionResourceClient createInstanceRelationshipsClient(RoutingCont
     return relatedInstancesClient;
   }
 
+  protected CollectionResourceClient createPrecedingSucceedingTitlesClient(RoutingContext routingContext, WebContext context) {
+    CollectionResourceClient relatedInstancesClient = null;
+    try {
+      OkapiHttpClient okapiClient = createHttpClient(routingContext, context);
+      relatedInstancesClient
+        = new CollectionResourceClient(
+        okapiClient,
+        new URL(context.getOkapiLocation() + "/preceding-succeeding-titles"));
+    } catch (MalformedURLException mfue) {
+      log.error(mfue);
+    }
+    return relatedInstancesClient;
+  }
+
+  protected String createQueryForPrecedingSucceedingInstances(List<String> instanceIds) {
+    String idList = instanceIds.stream().map(String::toString).distinct().collect(Collectors.joining(" or "));
+    return format("query=succeedingInstanceId==(%s)+or+precedingInstanceId==(%s)", idList, idList);
```

Comment:
Are the `+` symbols manual URL encoding?

If so, please change this to use the standard URL encoder.